README.md

```diff
@@ -27,3 +27,7 @@ pip install -r requirements.txt
 cd src
 python2.7 -m main.py
 ```
+
+## How to use Engine Application
```

Comment:
It is not Engine Application anymore. Its a game. Because the title of the particular game is quite ugly I would suggest to call this section "Features"